src/solvers/dgmulti/sbp.jl

```diff
@@ -311,23 +313,38 @@ function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS}) where {NDIMS}
   EToV = NaN # StartUpDG.jl uses size(EToV, 1) for the number of elements, this lets us reuse that.
   FToF = []
 
-  xyz = xyzq = rd.rst # TODO: DGMulti; extend to mapped domains
+  xyz = xyzq = rd.rst
   xyzf = ntuple(_ -> [], NDIMS)
   wJq = diag(rd.M)
 
   # arrays of connectivity indices between face nodes
   mapM = mapP = mapB = []
 
   # volume geofacs Gij = dx_i/dxhat_j
-  # TODO: DGMulti; extend to mapped domains
+  coord_diffs = coordinates_max .- coordinates_min
+
+  # Periodic SBP operators do not include one endpoint. We account for this by adding
+  # `h`` when estimating `factor`, which is the size of the "reference interval".
+  D = rd.approximationType
+  x = grid(D)
+  h = x[2] - x[1]
+  factor = (x -> x[2] - x[1])(extrema(grid(D))) + h
```

Comment:
That will work for FD methods on uniform meshes (constructed via `periodic_derivative_operator`) but not for general operators which might have non-uniform nodes, e.g., near the boundary. Let me introduce another API in SummationByPartsOperators.jl: Quarying `xmin` and `xmax` used when constructing the operators. Then, we can use this to map every SBP operator to the domain `[-1, 1]` when constructing the `RefElementData` as we do for non-periodic operators at the moment.

Reply:
Should be ready given that #1015 is merged